source/plugins/go/input/lib/cadvisor.go

```diff
@@ -253,6 +253,7 @@ func GetMetrics(winNode map[string]string, namespaceFilteringMode string, namesp
 
 func GetMetricsHelper(metricInfo map[string]interface{}, winNode map[string]string, namespaceFilteringMode string, namespaces []string, metricTime string) []metricDataItem {
 	var metricDataItems []metricDataItem
+	osType := strings.TrimSpace(os.Getenv("OS_TYPE"))
```

Comment:
we already have a osType in init. Maybe we can make it global and move this logic there?